build.gradle

```diff
@@ -55,8 +55,10 @@ compileTestJava {
 
 ext {
 	assertjVersion = '3.4.1'
+	jacksonVersion = '2.6.7'
+	log4jVersion = '2.6.2'
```

Comment:
Why do we need `Log4J2`?
The Jackson requirements?

There is no any commit comments on the matter.
Thanks